Dockerfile

```diff
@@ -1,7 +1,7 @@
-FROM php:7.3.11-fpm-alpine3.10
+FROM php:7.3.14-fpm-alpine3.11
```

Comment:
I am asking for more guidance on this, but I see latest Alpine version on DockerHub is 3.11.3. My assumption from https://hub.docker.com/_/alpine is that using 3.11 will automatically pull the latest 3.11.x version, which in this case is 3.11.3. Wouldn't it be better to be more explicit with the version # and explicitly request 3.11.3 so it's more deterministic which version we'll be getting with each build?

Reply:
Since we are using php's image we are at the mercy of that image which if you look in the source image it's pulling in 3.11 and not the explicit version so not much we can do short of rolling our own php and I'm not sure we want to be doing that right now.

Reply:
Understood, sorry I misread and thought we might have independent control of that. Too bad PHP isn't more specific in their versioning.